src/tox/config/sets.py

```diff
@@ -205,6 +209,7 @@ def work_dir_post_process(value: Path) -> Path:
             keys=["temp_dir"],
             of_type=Path,
             default=lambda conf, _: cast(Path, self["work_dir"]) / ".tmp",  # noqa: U100, U101
+            post_process=temp_dir_post_process,
```

Comment:
No, not post process 😆 it needs to be setup here https://github.com/tox-dev/tox/blob/main/src/tox/tox_env/api.py#L290 so e.g. for `tox config -e py` is not created 👍

Reply:
Should it be in `_setup_env` or in a separate function?

Reply:
Either is fine by me 😊